Markup kadai 11

blue2wo opened the pull request:
練習課題11「Google検索結果」1st commit
c590dfa HTML CSS

piasoratech commented on markup_kadai_11/markup_kadai_11.html:
```diff
@@ -0,0 +1,207 @@
+        <link rel="stylesheet" href="styleE11.css">
+    </head>
+<body>
+    <nav>
```
ここは`nav`タグを使用する場所ではないです。
ウェブサイト内の他のページへのリンクや、ページ内リンクなどの部分で使います

piasoratech commented on markup_kadai_11/markup_kadai_11.html:
```diff
@@ -0,0 +1,207 @@
+    </main>
+
+    <footer class="detailsnav">
+        <div id="footerinner">
```
ここは`div`タグを使用しないで実現してみましょう

piasoratech commented on markup_kadai_11/styleE11.css:
```diff
@@ -0,0 +1,327 @@
+
+nav{
+    flex: 1;
+/*    background-color: #fff;*/
```
不要なコードは含めないようにしましょう！

blue2wo commented:
e42458a
- removed unnecessary code and other kadai files. Only kadai 11 files remain.
- changed `<nav>` and `<footer>` tags

blue2wo commented:
練習課題11「Google検索結果」3rd push - feedback changes

56cd23d
- corrected errors based on https://validator.w3.org/nu/
- added `lang=ja`
- added `alt` tag to `<img>`
- changed `name` to `for` in `<label>`. `<label name="googlesearch"></label>` to `<label for="googlesearch"></label>`
- removed `border=0` from `<input>`

fjtawork commented on markup_kadai_11/markup_kadai_11.html:
```diff
@@ -0,0 +1,139 @@
+        </div>
+        </div>
+    </main>
+    <div class="detailsnav">
+        <div id="footerinner">
```
以前の指摘にもある通り、divよりもfooterタグのほうがいいですね！
